tests/e2e/specs/analysis.cy.js

```diff
@@ -16,16 +16,19 @@
  */
 
 describe('Analysis view', () => {
-  it('Should display the mocked workflow', () => {
+  beforeEach(() => {
     cy.visit('/#/analysis/one')
+  })
```

Comment:
FYI, the intention was to support other workflows in the offline data, which we may still do to help test views for which the simple example isn't enough. This view is a good example of one which could benefit from a more advanced example.

It's also not a bad idea to test a views interaction with the workspace page as we will add more interactions between views in due course. E.G. we already have the ability to open log files in a new tab, we should also acquire the ability to select a task in one view and use that to find the same task in another view (e.g. click+drag type functionality), keyboard controls, that sort of stuff.

No need to go collecting `cy.visit`s in all tests is the point, we may end up reversing these diffs.